testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/AbstractTestHiveViews.java

```diff
@@ -167,6 +168,45 @@ public void testSelectOnViewFromDifferentSchema()
         assertViewQuery("SELECT * FROM test_schema.hive_test_view_1", queryAssert -> queryAssert.hasRowsCount(25));
     }
 
+    @Test(groups = HIVE_VIEWS)
+    public void testViewReferencingTableInDifferentSchema()
+    {
+        String schemaX = "test_schema_x" + randomTableSuffix();
```

Comment:
This is unused, did you mean to put the view in this schema instead of the default one?

Reply:
yep i forgot to use it..